.gitmodules

```diff
@@ -4,3 +4,4 @@
 [submodule "lib/tinlake"]
 	path = lib/tinlake
 	url = https://github.com/centrifuge/tinlake
+	branch = develop
```

Comment:
It was not enough to just add this line here. It was still `master` branch. 

`dapp update` and `dapp install` are using gitmodules under the hood.

I manually removed the tinlake lib and re-installed it with a specific branch.

```bash
# remove sub module
git rm -r lib/tinlake
# install submodule with specific branch
git submodule add -b develop https://github.com/centrifuge/tinlake lib/tinlake
```

Information about submodules are stored in `.git` folder. It might be the case that everybody has to execute the commands as well.